labelbox/data/metrics/iou.py

```diff
@@ -0,0 +1,290 @@
+# type: ignore
+from typing import Dict, Any, List, Optional, Tuple, Union
+from shapely.geometry import Polygon
+from itertools import product
+import numpy as np
+
+from labelbox.data.metrics.preprocess import label_to_ndannotation
+from labelbox.schema.bulk_import_request import (NDAnnotation, NDChecklist,
+                                                 NDClassification, NDTool,
+                                                 NDMask, NDPoint, NDPolygon,
+                                                 NDPolyline, NDRadio, NDText,
+                                                 NDRectangle)
+from labelbox.data.metrics.preprocess import (create_schema_lookup,
+                                              url_to_numpy)
+
+VectorTool = Union[NDPoint, NDRectangle, NDPolyline, NDPolygon]
+ClassificationTool = Union[NDText, NDRadio, NDChecklist]
+
+
+def mask_iou(predictions: List[NDMask], labels: List[NDMask]) -> float:
+    """
+    Creates prediction and label binary mask for all features with the same feature scheama id.
```

Comment:
Does this method check for same feature schema id's, or is that something assumed from the input?

Reply:
It is not checked here. Technically it could accept any ids. because it is just calculating the iou between any set of masks.